source/includes/_groups.md

```diff
@@ -122,7 +122,7 @@ This endpoint retrieves a specific group.
 
 ### HTTP Request
 
-`GET https://public-api.lumiformapp.com/api/v2/group/<GroupId>`
+`GET https://public-api.lumiformapp.com/api/v2/groups/<GroupId>`
```

Comment:
This and similar fixes are needed because we actually don't have the singular version of the endpoint. It's always plural